### DISCUSSION TITLE
Support for SQLite?

### DISCUSSION THREAD

**Discussion by ad-si** [ToolJet/ToolJet#3624] [Q&A]
Does ToolJet support querying data from a SQLite database?

**Comment by withshubh** [comment]
Hey @ad-si 👋 

Unfortunately, we don't have the datasource available for SQLite.

It would be great if you could create a feature request issue [here](https://github.com/ToolJet/ToolJet/issues/new?assignees=&labels=feature+request%2C+needs+triage&template=02-feature_request.md). We might consider adding it in the future.

Thanks!
